napari/components/viewer_model.py

```diff
@@ -580,28 +580,55 @@ def _calc_status_from_cursor(
     ) -> tuple[str | Dict, str] | None:
         if not self.mouse_over_canvas:
             return None
+        coord2val: dict[str, list[str]] = {}
+        coord_str = ''
+        status_str = ''
+        tooltip_text = ''
         active = self.layers.selection.active
-        if active is not None and active._loaded:
-            status = active.get_status(
+
+        for layer in self.layers[::-1]:
+            if not layer.visible or layer.opacity == 0 or not layer._loaded:
+                continue
+            status = layer.get_status(
                 self.cursor.position,
                 view_direction=self.cursor._view_direction,
                 dims_displayed=list(self.dims.displayed),
                 world=True,
             )
-
-            if self.tooltip.visible:
-                tooltip_text = active._get_tooltip_text(
-                    np.asarray(self.cursor.position),
-                    view_direction=np.asarray(self.cursor._view_direction),
-                    dims_displayed=list(self.dims.displayed),
-                    world=True,
+            emphasis = ' ### ' if layer is active else ''
```

Comment:
Much smaller, and looks nice and set out still in my eye
```suggestion
            emphasis = ' ** ' if layer is active else ''
```

Reply:
I actually had the stars before but my eyes struggled to find them...